### DISCUSSION TITLE
Question about the text widget.

### DISCUSSION THREAD

**Discussion by ghost** [templaza/astroid-framework#538] [General]
Is it correct that only text is possible in the text widget? I wanted a link in the text, but that doesn't work for me. Where the linked word is in the text, <a href="link target">link text</a> appears

Am I making a mistake or is it really just plain text?

That would be a shame, because sometimes it is very useful if you can put a link in the text.

Regards Olli

**Comment by sonvnn** [comment]
Good news! In Astroid 3.0.9 text widget is using editor. So you can insert link or any format easily. Please update Astroid 3.0.9

**Comment by ghost** [comment]
Hello Sonny, thank you very much for your answer. I use Android 3.0.9 and tried to copy and paste the links into the text field. That does not work. So I have to work with the insert link button in the editor?
Regards Olli

**Reply by sonvnn** [reply]
Please clear browser cache and refresh it again. Thanks,

**Comment by ghost** [comment]
Hello, thank for taking the time and trying to help me. I think they didn't understand me properly because my English is so bad.

Inserting a link using the editor button works, that's not my problem.

I'm trying to add a link using the keyboard like:

`<a href="tel:+49xxx..."><i class="fa-solid fa-phone"></i> <span style="text-decoration: underline;">+49 xxxx / xxxx...</span></a><br /> <i class="fa-solid fa-envelope"></i>`

This does not work for me. In the frontend I don't see the link with Fontawesome and phone number but rather the code entered.

Is the Text Widget not suitable for this or am I making a mistake?

Regards Olli
